app/models/job.ts

```diff
@@ -87,6 +85,106 @@ export interface JobQuery {
   }
 }
 
+// State machine definition for jobs. This is not used to maintain state, just to enforce
+// transition rules
+const stateMachine = createMachine(
+  {
+    id: 'job',
+    initial: 'accepted',
```

Comment:
NOTE: We don't ever actually use the 'ACCEPTED' status outside of tests. Jobs are always started as 'RUNNING' or 'PREVIEWING'. We should revisit whether or not we need this anymore.